samples/jvm/junit5/.gitignore

```diff
@@ -1,10 +1,11 @@
 # Ignore Gradle project-specific cache directory
-.gradle
+/.gradle
```

Comment:
out of curiosity: unless I am wrong, this will only ignore .gradle in the root, why do you want to commit it if it is not in the root?

Reply:
Ah, I see.  I'll back this out.  I was working from this dir as-if it were project root